docs/faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md

```diff
@@ -43,20 +45,30 @@ Some tools fail to calculate duplication or time out when analyzing a number of
 See <a href="../../troubleshooting/why-arent-duplication-metrics-being-calculated/">Why aren't duplication metrics being calculated?</a>
 </td>
 </tr>
+
 <tr>
 <td>Number of issues per file and per tool</td>
 <td>50</td>
 <td>
 Codacy limits the number of issues returned on each file by individual tools as a safeguard against degradation of performance on large or unexpected use cases. This means that in some situations Codacy could report more issues after a push that includes fixes for the currently reported issues.
 </td>
 </tr>
+
 <tr>
 <td>Number of comments on the Git provider</td>
 <td>25</td>
 <td>
 Codacy limits the number of comments for reporting found issues on pull requests to avoid triggering too many notification emails and to guard against hitting API rate limits.
 </td>
 </tr>
+
+<tr>
+<td>Showing issues on duplicated lines</td>
+<td>-</td>
+<td>
+For now, Codacy only shows the first code issue when there are issues on duplicated lines over the same file. 
```

Comment:
Two minor suggestions:

```suggestion
For now, Codacy only reports the first code issue when there are issues on duplicated lines on the same file. 
```

I think we can say "reports" instead of "shows" to include for example the results that the Codacy API provides, email notifications, status checks, etc.